components/openindiana/qemu-kvm/Makefile

```diff
@@ -18,12 +18,12 @@ include ../../../make-rules/shared-macros.mk
 
 PATH=	/opt/onbld/bin/i386:/usr/bin:/usr/sbin:/sbin:/usr/gnu/bin
 COMPONENT_NAME=		illumos-kvm-cmd
-COMPONENT_VERSION=	c1a197c8e4582c68739ab08f7e3198b2392c9820
-COMPONENT_REVISION=	2
-IPS_COMPONENT_VERSION=	0.0.1.20160303
+COMPONENT_VERSION=	41e7c3e192eb39bb5ced7667fdc12ad2b33e646d
+COMPONENT_REVISION=	3
```

Comment:
I wonder whether it we can drop COMPONENT_REVISION here as the package will get completely new version information anyways.

Reply:
yes, we can

Reply:
IIRC a git revision should break IPS's update behavior. In my research it is explicitly mentioned that the solver tries to work with Semantic versions in the main. When we use incorporations, then it won't matter but without we still need some version number that follows semantic versioning.

Reply:
The actual FRMI is pkg://openindiana.org/system/qemu/kvm@0.0.1.20160303-2018.0.0.2:20180912T131705Z
The package will get a new Component Version (0.0.1.20190827) and a new Branch Version (either 2020.0.0.3 or 2020.0.0.0 depending on the COMPONENT_REVISION).

Reply:
I did forget to remove this line after bumping to the 2019 tag.

Reply:
Fixed